src/umpire/strategy/AllocationStrategy.hpp

```diff
@@ -12,9 +12,13 @@
 #include <ostream>
 #include <string>
 
+#include "camp/camp.hpp"
+#include "camp/resource.hpp"
 #include "umpire/util/MemoryResourceTraits.hpp"
 #include "umpire/util/Platform.hpp"
 
+using Resource = camp::resources::Resource;
```

Comment:
This looks unused.

Reply:
Good catch, removed.

Reply:
It is actually being used by some other files that have included this header - so I also used it in the allocate/deallocate_resource functions at the bottom of the header

Reply:
Please don't pollute global namespace.